source/documents/view.txt

```diff
@@ -113,10 +113,9 @@ Expand Embedded Objects and Arrays
          :figwidth: 696px
          :alt: View nested elements in Table view
 
-      The following example displays the ``delays``
-      object nested within the ``airlines`` collection documents. The
-      new tab displays the two properties of the ``delays`` object:
-      ``international`` and ``domestic``.
+      The following example displays the ``address``
+      object nested within the ``restaurants`` collection documents. The
+      new tab displays the five properties of the ``address`` object:
 
       .. figure:: /images/compass/doc-table-view-nested.png
```

Comment:
(suggestion -- existing content)
The encrypted fields image in the List View is sized differently from every other image on this page. It might be worth adjusting, while you're here.

Reply:
I made the image smaller so that it aligns better with the rest of the images but is still large enough to show the encrypted field